### DISCUSSION TITLE
vk.com visited from ruby:alpine doesn't show charset properly

### DISCUSSION THREAD

**Discussion by Nakilon** [rubycdp/ferrum#363] [Q&A]
How do I fix this?
![image](https://github.com/rubycdp/ferrum/assets/2870363/21d24194-96ac-4acd-bff0-3296ae3a42f8)
I guess I only used ferrum for English content and so didn't notice such issue. Was sure page html code handles it... let me see:
![image](https://github.com/rubycdp/ferrum/assets/2870363/d0b4511a-f453-41e9-ac87-0672f7caf82e)
Should I apt install some cp1251-related package?

EDIT: sry, not Ubuntu. docker ruby:alpine

**Comment by route** [comment]
Yea I think it's missing locale on the OS or misconfiguration
